cli/cmd/auth/login.go

```diff
@@ -75,7 +75,9 @@ func Login(ctx context.Context, ch *cmdutil.Helper, redirectURL string) error {
 
 	ch.PrintfBold("\nOpen this URL in your browser to confirm the login: %s\n\n", deviceVerification.VerificationCompleteURL)
 
-	_ = browser.Open(deviceVerification.VerificationCompleteURL)
+	if ch.Interactive {
```

Comment:
Perhaps would be good to wrap the browser.Open? Like `ch.Open`? Would be good to make it a standard in all usages. There are a lot more places where we call `browser.Open`

Reply:
I see the motivation, but at the moment, I'm hesitant to do that since commands like `rill docs` and `rill sudo user open` are explicitly meant to open the browser. For now, I think it's fine as is.